### DISCUSSION TITLE
Data Export and Inhomogeneous Meshing Issues

### DISCUSSION THREAD

**Discussion by yangyanghei** [damask-multiphysics/DAMASK#21] [Q&A]
I hope this message finds you well. I am currently facing several challenges while using the DAMASK code and would greatly appreciate your assistance in resolving them.
﻿
Firstly, I require guidance on how to export simulation results for a specific element. Could you please provide instructions on how to achieve this?
﻿
Secondly, I am utilizing the spectral solver for my simulations. However, I am facing limitations in grid partitioning, specifically in refining the mesh near grain boundary regions. This refinement is crucial for achieving higher accuracy in my results. However, the inhomogeneous meshing appears to restrict the use of spectral solvers. Can you please advise on how to implement an inhomogeneously distributed mesh model and successfully perform simulations using DAMASK?
﻿
Any advice, documentation references, or tutorials related to these issues would be greatly appreciated to help me progress effectively with my simulations.
﻿
Thank you very much for your attention to these matters. I look forward to your prompt response.

**Comment by MarDiehl** [comment]
You csn use damask.Result.place to get spatially orderered results
